Makefile

```diff
@@ -64,7 +65,7 @@ coverage: venv
 
 test: venv
 	@echo "Executing functional tests..."
-	@. $(ACTIVATE); py.test $(TEST_DIRS)/functional/
+	@. $(ACTIVATE); py.test $(TEST_DIRS)/functional/ --test_dir $(TEST_MODELS_DIR)
 
 test_local_only: venv
 	@echo "Executing functional tests with only local models..."
```

Comment:
now also those test can be added as local only:
tests/functional/test_model_version_policy.py
tests/functional/test_model_versions_handling.py
tests/functional/test_reshaping.py